data/presets/fields.json

```diff
@@ -951,6 +951,11 @@
             "type": "check",
             "label": "Indoor"
         },
+        "indoor": {
```

Comment:
I think this is a mistake, should be "industrial"..
(I don't usually edit the `fields.json` file, it's autogenerated when doing a local build.  So even if we did merge it this way, it would get fixed automatically anyway)